core/payment-driver/zksync/src/driver.rs

```diff
@@ -175,8 +175,14 @@ impl PaymentDriver for ZksyncDriver {
         _caller: String,
         msg: Enter,
     ) -> Result<String, GenericError> {
-        log::info!("ENTER = Not Implemented: {:?}", msg);
-        Ok("NOT_IMPLEMENTED".to_string())
+        let tx_hash = wallet::enter(&msg).await?;
+
+        Ok(format!(
+            "Deposit transaction has been sent on Ethereum and soon funds should be available \
+        on ZkSync network. You can check transaction status in the block explorer. \
+        Tracking link: https://rinkeby.zkscan.io/explorer/transactions/{}",
```

Comment:
This link won't work for mainnet deposits.

Reply:
What would be the best option? The same way tracking link is provided in `exit` command.
Can we leave it, with opening an issue to fix it the proper way. :thinking: rinkeby link is still better than no link?
WDYT?

Reply:
OK, leave it for now.
@maaktweluit Plz add a UX task to the core team backlog to fix the tracking links for mainnet.

Reply:
done: https://github.com/golemfactory/yagna/issues/1251